packages/coinstac-ui/app/render/components/runs/effects/useStartInitialRuns.jsx

```diff
@@ -0,0 +1,47 @@
+import { useRef, useEffect } from 'react';
+import { useLazyQuery, useQuery } from '@apollo/client';
+import { get } from 'lodash';
+import { useDispatch } from 'react-redux';
+
+import { FETCH_ALL_USER_RUNS_QUERY, FETCH_ALL_CONSORTIA_QUERY } from '../../../state/graphql/functions';
+import { startRun } from '../../../state/ducks/runs';
+
+/*
+ * This effect checks if there are runs that must be started locally when the app first starts.
```

Comment:
whats the differences between this func and the above? Can we communicate that preferably with naming or as an alternative comments?

Reply:
This one just runs on startup. It's responsible for checking decentralized runs that did not start in the client yet.

The above one listens for a subscription called `runStarted` that is pushed when a new decentralized run is created in the api.

I separated both thinking it would be easier to understand/maintain.

Reply:
The comment on both files says `when the app first starts` can we clarify this?